webpack.config.js

```diff
@@ -10,22 +10,33 @@ module.exports = {
     'react': 'React',
     'react/addons': 'React'
   },
+  fallback: path.join(__dirname, "node_modules"),
   module: {
     loaders: [
       {
         test: /\.js$/,
         loaders: ['babel-loader'],
-        include: [path.resolve(__dirname, 'node_modules/webmaker-core/src'), path.resolve(__dirname, 'src')]
+        include: [
+          /\/webmaker-core\//,
```

Comment:
Alternative solution to @toolness's in https://github.com/mozilla/webmaker-browser/pull/153

I'm happy to do either